parse-facebook-user-session.js

```diff
@@ -53,10 +53,9 @@ var parseFacebookUserSession = function(params) {
     Parse.Promise.as().then(function() {
       // Make a request object. Its objectId will be our XSRF token.
       maybeLog("Creating ParseFacebookTokenRequest...");
-      var url = 'https://' + req.host + req.path;
       var request = new Parse.Object("ParseFacebookTokenRequest");
       return request.save({
-        url: url,
+        url: req.url.slice(1),
```

Comment:
Need to slice the "/" off the url since express rewrites urls to be relative to the mount point. Redirects interpret the leading "/" as being relative to the domain rather than relative to the mount point. Annoying, but this ensures both sides treat this value as relative to the mount point.